js/index.js

```diff
@@ -289,26 +458,41 @@ document.addEventListener("DOMContentLoaded", () => {
   //2. Showing skills section
   showSkills();
 
-  //3. Showing copyright footer
+  //3. Showing projects section
+  let method = "GET";
+  let url = "https://api.github.com/users/secch97/repos?sort=created&direction=asc";
+  //3.1 Creating the request object:
+  let apiRequestObject = new apiRequest(method, url);
+  //3.2 Setting the request object:
+  apiRequestObject.setRequest = new XMLHttpRequest();
+  //3.3 Getting the request object:
+  let projectsRequest = apiRequestObject.getRequest;
+  //3.4 Sending the request:
+  projectsRequest.send();
+  //3.5 Adding load event to request. Once loaded, run the handleProjectsData function:
+  projectsRequest.addEventListener("load", handleProjectsData);
```

Comment:
Let's add the event listener before we call `send`.

Reply:
I will make sure to do that